README.md

```diff
@@ -1,9 +1,9 @@
 # IEKit
-An autonomous service implementing a decentralized Impact Evaluator built with the [Open Autonomy](https://docs.autonolas.network/open-autonomy/) framework. The service tracks mentions to [@autonolas](https://twitter.com/autonolas) on Twitter, assigns scores to them and writes those scores to a [Ceramic](https://ceramic.network/) stream. It then reads verified wallets (a mapping between Wwitter handles and Ethereum addresses) from another Ceramic stream, checks for users that have minted a token, and updates their token metadata according to their scores so NFT images are upgraded over time.
+An autonomous service implementing a decentralized Impact Evaluator built with the [Open Autonomy](https://docs.autonolas.network/open-autonomy/) framework. The service tracks mentions to [@autonolas](https://twitter.com/autonolas) on Twitter, assigns scores to them and writes those scores to a [Ceramic](https://ceramic.network/) stream. It then reads verified wallets (a mapping between Twitter handles and Ethereum addresses) from another Ceramic stream, checks for users that have minted a token, and updates their token metadata according to their scores so NFT images are upgraded over time.
```

Comment:
Should be "mentions of" rather than "mentions to"

Reply:
Last sentence could be clearer as an ordered list.

```
It then:
1. reads verified wallets (a mapping between Twitter handles and Ethereum addresses) from another Ceramic stream,
2. checks for users that have minted a token, and
3. updates their token metadata according to their scores so NFT images are upgraded over time.
```

Reply:
fixed